umh.docs.umh.app/content/en/docs/concepts/security/management-console.md

```diff
@@ -0,0 +1,76 @@
+---
+title: "Management Console"
+content_type: concept
+description: |
+    Concepts related to the security of the Management Console.
+weight: 10
+---
+
+<!-- overview -->
+The web-based nature of the Management Console means that it is exposed to the
+same security risks as any other web application. This section describes the
+measures that we adopt to mitigate these risks.
+
+<!-- body -->
+
+## Encrypted Communication
+
+The Management Console is served over HTTPS, which means that all communication
+between the browser and the server is encrypted. This prevents attackers from
+eavesdropping on the communication and stealing sensitive information such as
+passwords and session cookies.
+
+## Cyphered Messages
+
+{{% notice warning %}}
+This feature is currently in development and is subject to change.
+{{% /notice %}}
+
+Other than the standard TLS encryption provided by HTTPS, we also provide an
+additional layer of encryption for the messages exchanged between the Management
+Console and your UMH instance.
+Every action that you perform on the Management Console, such as creating a new
+data source, and every information that you retrieve, such as the messages in
+the Unified Namespace, is encrypted using a secret key that is only known to
+you and your UMH instance. This ensures that no one, not even us, can see, read
+or reverse engineer the content of these messages.
+
+The process we use (which is now patent pending) is simple yet effective:
+
+1. When you create a new user on the Management Console, we generate a new
+   private key and we encrypt it using your password. This means that only you
+   can decrypt it.
+2. The encrypted private key and your hashed password are stored in our database.
+3. When you login to the Management Console, we verify your password and check
```

Comment:
When you login into the Management Console, you will receive the encrypted private key, and need to be able to decrypt it in order to successfully login (it is technically more complicated, but in short your password never reaches our servers)